test/client.test.ts

```diff
@@ -861,10 +861,13 @@ test("fetch: tuple", async () => {
     expect(t0.length).toBe(2);
     expect(JSON.stringify(t0)).toBe('[1,"abc"]');
 
-    const insp = util.inspect(t0);
-    expect(
-      insp === "Tuple [ 1, 'abc' ]" || insp === "Tuple(2) [ 1, 'abc' ]"
-    ).toBeTruthy();
+    if (!isDeno) {
```

Comment:
This should be factored into a function like `expectInspectToBe()` which would do nothing on Deno